docs/faq/general.txt

```diff
@@ -71,6 +71,24 @@ Django was originally developed at World Online, the Web department of a
 newspaper in Lawrence, Kansas, USA. Django's now run by an international
 `team of volunteers <https://www.djangoproject.com/foundation/teams/>`_.
 
+How is Django licensed?
+=======================
+
+Django is distributed under `the BSD license
```

Comment:
I would say "the 3-clause BSD license" or else "a BSD license," since there is more than one BSD license:
https://opensource.org/licenses/BSD-3-Clause